SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -5509,23 +5541,23 @@ CDiscAdjFSIDriver::CDiscAdjFSIDriver(char* confFile,
   direct_iteration = new CIteration*[nZone];
 
   unsigned short iZone;
-  for (iZone = 0; iZone < nZone; iZone++){
-    switch (config_container[iZone]->GetKind_Solver()) {
-      case DISC_ADJ_INC_RANS: case DISC_ADJ_INC_EULER: case DISC_ADJ_INC_NAVIER_STOKES:      
-       case DISC_ADJ_RANS: case DISC_ADJ_EULER: case DISC_ADJ_NAVIER_STOKES:
-         direct_iteration[iZone] = new CFluidIteration(config_container[iZone]);
-         nVar_Flow = solver_container[iZone][INST_0][MESH_0][ADJFLOW_SOL]->GetnVar();
-         flow_criteria = config_container[iZone]->GetMinLogResidual_BGS_F();
-         flow_criteria_rel = config_container[iZone]->GetOrderMagResidual_BGS_F();
-         break;
-       case DISC_ADJ_FEM:
-         direct_iteration[iZone] = new CFEAIteration(config_container[iZone]);
-         nVar_Struct = solver_container[iZone][INST_0][MESH_0][ADJFEA_SOL]->GetnVar();
-         structure_criteria    = config_container[iZone]->GetMinLogResidual_BGS_S();
-         structure_criteria_rel = config_container[iZone]->GetOrderMagResidual_BGS_S();
-         break;
-    }
-  }
+//  for (iZone = 0; iZone < nZone; iZone++){
```

Comment:
For primal FSI problems it uses the new Multizone driver implementation now. Check the test cases. The output has changed, and consequently also the regression test values. But I checked it very carefully and made that they give identical results.

Reply:
Ok, should dead code be removed then? Not just for this driver that is about to be made redundant but for other areas that you may have commented out (I do not know if there are any).

I ask this because commented code is not possible to maintain as it does not trigger warnings or errors. For example with the work of CVariable I am sure I missed some areas that will probably not compile if commented in...

Reply:
I am chiming in unannounced, but I am a big proponent of removing commented out code.